File: scala/src/test/scala/com/shekhargulati/ninetynine_problems/_02_arithmetic/P38Test.scala
package com.shekhargulati.ninetynine_problems._02_arithmetic

import org.scalatest.{FunSpec, Matchers}

class P38Test extends FunSpec with Matchers {

  describe("P38 Spec") {
    it("35 and 64 are coprime") {
      val coprime: Boolean = P38.coprime(35, 64)
      coprime should be(true)
    }
  }

}

File: java8/src/test/java/com/shekhargulati/ninetynine_problems/_02_arithmetic/P38Test.java
package com.shekhargulati.ninetynine_problems._02_arithmetic;

import org.junit.Test;

import static org.junit.Assert.assertTrue;

public class P38Test {

    @Test
    public void shouldSay35And64IsCoprime() throws Exception {
        boolean coprime = P38.coprime(35, 64);
        assertTrue(coprime);
    }
}